service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ValidateOrderController.java

```diff
@@ -11,24 +11,26 @@
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.generatedorder.InterimEndDate;
 import uk.gov.hmcts.reform.fpl.service.ValidateGroupService;
 import uk.gov.hmcts.reform.fpl.validation.groups.epoordergroup.EPOAddressGroup;
 import uk.gov.hmcts.reform.fpl.validation.groups.epoordergroup.EPOEndDateGroup;
 
 import java.util.List;
+import javax.validation.groups.Default;
 
 import static uk.gov.hmcts.reform.fpl.enums.EPOType.PREVENT_REMOVAL;
+import static uk.gov.hmcts.reform.fpl.enums.ccd.fixedlists.InterimEndDateType.NAMED_DATE;
 
 @Api
 @RestController
-@RequestMapping("/callback/validate-emergency-protection-order")
-public class ValidateEmergencyProtectionOrderController {
+@RequestMapping("/callback/validate-order")
+public class ValidateOrderController {
     private final ObjectMapper mapper;
     private final ValidateGroupService validateGroupService;
 
     @Autowired
-    public ValidateEmergencyProtectionOrderController(ObjectMapper mapper,
-                                                      ValidateGroupService validateGroupService) {
+    public ValidateOrderController(ObjectMapper mapper, ValidateGroupService validateGroupService) {
```

Comment:
@Gogetter showed me a nice way of doing this using lombok RequestArgs, perhaps we could use it here too since we're changing it?

```
@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Reply:
Yeah, although as it is a service technically we don't even need `@Autowired`. I generally only  add `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` when the constructor is starting to be a bit too large.